test/solc/Common.cpp

```diff
@@ -54,5 +54,22 @@ test::OptionsReaderAndMessages test::parseCommandLineAndReadInputFiles(
 	if (success && _processInput)
 		success = cli.processInput();
 
-	return {success, cli.options(), cli.fileReader(), cli.standardJsonInput(), sout.str(), serr.str()};
+	return {
+		success,
+		cli.options(),
+		cli.fileReader(),
+		cli.standardJsonInput(),
+		sout.str(),
+		stripPreReleaseWarning(serr.str()),
+	};
+}
+
+string test::stripPreReleaseWarning(string const& _stderrContent)
```

Comment:
Do we know the string we're getting is **always** of type `std::string`? If not, maybe use `std::string_view` instead to avoid unnecessary expensive object constructions.

Reply:
Actually, the string goes right into `regex_replace()` and that requires a string so I think there's no point taking a `string_view` only to convert it to string immediately.